README.md

```diff
@@ -1 +1,27 @@
-# oop-school-library
\ No newline at end of file
+# oop-school-library
+> In this project we are building school library app. At this phase of the project, classes that models students and teachers are implemented. 
+
+## Built With
+
+- **Ruby**
+
+## Getting Started
+
+### Steps to run the project
+
+#### 1 - Install Ruby.
+
+#### 2 - Clone the repository.
+```
+git clone https://github.com/belmeetmule/oop-school-library.git
+```
+
+#### 3 - run the project
+> Try it out in your IRB console by instantiating any of the classes
+
+## Author
+### 👨‍💻 Mulugeta M.
+
+- GitHub: [@belmeetmule](https://github.com/belmeetmule)
+- Twitter: [@belmeetmule](https://twitter.com/belmeetmule)
+- LinkedIn: [belmeetmule](https://linkedin.com/in/belmeetmule)
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/201909084-fe1fefdf-2d02-4762-8a2b-bd0099de0035.png)

- [ ] You didn't add these sections in your readme file shown in the image above :arrow_up: Kindly add the missing section using this [microverse readme template](https://github.com/microverseinc/readme-template#-contributing) :+1: